projects/go-lib/src/lib/components/go-search/go-search.component.ts

```diff
@@ -75,6 +80,18 @@ export class GoSearchComponent implements OnInit {
     }
   }
 
+  keyDown(event: Partial<KeyboardEvent>): void {
+    if (event.code === 'ArrowDown') {
+      this.goSearchService.arrowDownEvent.emit();
+    }
+  }
```

Comment:
Should we allow for more keyboard navigation than just arrow key down here? If we're looking for navigation within search results the user should have the ability to go down or up in the results right? Should we also add the ability to close the search with the escape key? Not sure if that works currently, but I would assume it doesn't.

Reply:
See my comment below, but we could look at doing the escape key as well.